script/generate

```diff
@@ -174,7 +175,8 @@ generate()
     # Define generation variables
     local slcp=${1?A .slcp file is expected as the first argument}
     local project_name=""
-    project_name=$(grep -oP 'project_name: \K(.*)' "${slcp}")
+    project_line=$(grep 'project_name: ' "${slcp}")
+    project_name="${project_line//project_name:/}"
```

Comment:
```suggestion
    project_name="${project_line//project_name: /}"
```

Reply:
I had that first, but the substitution already removes the spaces as well and I didnt know if I could always count on a single space int the slcp file

Reply:
Hmm, that must be a zsh vs. bash issue. I'll check it

Reply:
OK, I put in a separate step to strip all spaces so the script can handle someone hand editing sclp or spacing changes in there